docs/reference/api/pages.md

```diff
@@ -0,0 +1,511 @@
+      "level": 1,
+      // ...
+      "_url": "http://localhost:3000/blog",
+      "_children": []
+    },
+    {
+      "_id": "ckhurshqd00088v9k6pfnqpjz",
+      "visibility": "public",
+      "type": "default-page",
+      "title": "About us",
+      "slug": "/about-us",
+      "rank": 1,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhurshqd00088v9k6pfnqpjz",
+      "level": 1,
+      // ...
+      "_url": "http://localhost:3000/about-us",
+      "_children": [
+        "ckiz0pmsg006gdr9ker7e99yt",
+        "ckiz0pyym0070dr9kv8hzm7y0"
+      ]
+    },
+    {
+      "_id": "ckiz0pmsg006gdr9ker7e99yt",
+      "visibility": "public",
+      "type": "default-page",
+      "title": "Our history",
+      "slug": "/about-us/our-history",
+      "rank": 0,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhurshqd00088v9k6pfnqpjz/ckiz0pmsg006gdr9ker7e99yt",
+      "level": 2,
+      // ...
+      "_url": "http://localhost:3000/about-us/our-history",
+      "_children": []
+    },
+    {
+      "_id": "ckiz0pyym0070dr9kv8hzm7y0",
+      "visibility": "public",
+      "type": "default-page",
+      "title": "Leadership and staff",
+      "slug": "/about-us/leadership-and-staff",
+      "rank": 1,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhurshqd00088v9k6pfnqpjz/ckiz0pyym0070dr9kv8hzm7y0",
+      "level": 2,
+      // ...
+      "_url": "http://localhost:3000/about-us/leadership-and-staff",
+      "_children": []
+    },
+    {
+      "_id": "ckhdscx6000084z9k56lybljw",
+      "visibility": "public",
+      "type": "@apostrophecms/trash",
+      "title": "Trash",
+      "slug": "/trash",
+      // ...
+      "rank": 3,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhdscx6000084z9k56lybljw",
+      "level": 1,
+      "_children": []
+    }
+  ]
+}
+```
+
+## `GET: /api/v1/@apostrophecms/page/:id`
+
+### Request example
+
+```javascript
+// Request inside an async function.
+const document = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey', {
+  method: 'GET'
+});
+```
+
+### Response
+
+The successful GET request returns the matching document. See the [page document response example](#page-document-response-example) below for a sample response body.
+
+## `POST: /api/v1/@apostrophecms/page`
+
+**Authentication required.**
+
+### Required properties
+
+| Property | Type | Description |
+|----------|------|-------------|
+|`_targetId` | String | The `_id` of an existing page to use as a target when inserting the new page|
+|`_position` | String | `before`, `after`, `firstChild`, or `lastChild`. This sets the position within the page tree for the new page in relation to the target page (see `_targetId`). `before` and `after` insert the new page as a sibling of the target. `firstChild` and `lastChild` insert the new page as a child of the target.|
+
+The `_position` property uses specific string values rather than index numbers to better support the draft review workflow.
```

Comment:
This is no longer the case per your PR (: `_position` may be a number and that is perfectly fine and implies a child of `_targetId`. Or it can be one of the four string values.

Reply:
I think this comment did not get addressed?

Reply:
(Specifically, that index numbers *are* allowed, per your past PR)

Reply:
Bumping this.

Reply:
I did that? I did that.